src/distance/distance.cpp

```diff
@@ -0,0 +1,82 @@
+#include <cmath>
+#include <utility>
+#include <iomanip>
+
+#include "spatial/attitude-utils.hpp"
```

Comment:
Use `#include "common/spatial/attitude-utils.hpp"` and `#include "common/spatial/camera.hpp"`